compose-add-ons/ssh-forwarding.yml

```diff
@@ -0,0 +1,11 @@
+services:
+  devel:
+    environment:
+      - SSH_AUTH_SOCK=$SSH_AUTH_SOCK_CONTAINER_PATH
```

Comment:
I think, we need to be careful concerning user authorization using the `ssh-agent`

Reply:
Could you elaborate more on your concern?